frontend/src/pages/operation_list/operation_card/stylesheet.styl

```diff
@@ -54,6 +54,43 @@
   &:hover
     background-color: $darker-background
 
+.favorite
```

Comment:
My CSS skills aren't great, but I think we can reduce the duplication here.

1. Keep `.not-favorite` (consider a rename -- you'll see why in a bit)
2. Rename `favorite` and remove everything but `background-image: url('./star-solid.svg')`
3. in the operation_card/index.tsx, changed the classname as such:
`className={cx('not-favorite', isFavorite && 'favorite')}`

For what it's worth, I renamed these to `favorite-button` (with the unfilled star), and `filled`. Here's what my final result looks like:

```css
.favorite-button
  position: absolute
  top: 70px
  right: 15px
  width: 25px
  min-width: 25px
  height: @width
  background-color: $background
  background-position: center center
  background-size: 12px
  background-repeat: no-repeat
  background-image: url('./star-outline.svg')
  box-shadow: inset 0 0 0 1px $darker-background
  border-radius: 3px
  padding: 0px

  &.filled
    background-image: url('./star-solid.svg')

  &:hover
    background-color: $darker-background
```

```tsx
<Button
        className={cx('favorite-button', isFavorite && 'filled')}
        onClick={() => setIsFavorite(!isFavorite)}>
      </Button>
```

Reply:
oh duh - I'm so using to working with JS-within-css (like styled components) that I forgot how to de-dup css the actual way 😅